src/js/components/agency/v2/CountTab.jsx

```diff
@@ -13,28 +13,38 @@ const propTypes = {
     setActiveTab: PropTypes.func.isRequired,
     active: PropTypes.bool,
     count: PropTypes.number,
-    subCount: PropTypes.number
+    subCount: PropTypes.number,
+    disabled: PropTypes.bool
 };
 
-const CountTab = (props) => (
-    <button
-        className={`count-tabs__button${props.active ? ' count-tabs__button_active' : ''}`}
-        onClick={() => props.setActiveTab(props.type)}>
-        <div className="count-button">
-            <div className="count-button__label">
-                {props.label}
-            </div>
-            <div className="count-button__count">
-                {(props.count || props.count === 0) ? `${props.count}` : '--'}
-            </div>
-            {props.subHeading ? (
-                <div className="count-button__sub-heading">
-                    With {(props.subCount || props.subCount === 0) ? `${props.subCount}` : '--'} {props.subHeading}
+
+const CountTab = (props) => {
+    const setActiveTab = () => {
+        props.setActiveTab(props.type);
+    };
+
+    return (
+        <button
+            className={`count-tabs__button${props.active ? ' count-tabs__button_active' : ''}`}
+            onClick={setActiveTab}
+            disabled={props.disabled}
+            value={props.type}>
```

Comment:
We do not need value anymore since we are not dynamically setting the value of this button. That was an oversight on my part.

Reply:
Ok, I removed it. Thanks @jonhill13!